site/config.yaml

```
@@ -140,9 +142,12 @@ languages:
         - name: Help-Center
           url: https://help.skribble.com/english
           weight: 10
-        - name: QES explained
-          url: /qes-explained/
+        - name: Signaturstandards
+          url: /signaturestandards/
```

```suggestion
          url: /signature-standards/
```
What do you think of using a hyphen between words like we do for `qes-explained` and the like? For me it is easier to read if we put a `-` where there would normally be a space (` `) between words.

---

I leave it like this and Till looks at it again anyway. Then we can adjust it again.